_includes/header.html

```diff
@@ -13,7 +13,7 @@
     <a href="/" style='text-decoration: none; color:black;'>
       <div class="wrapper">
         <p>Department of</p>
-        <p>COMPUTER SCIENCE</p>
+        <p>COMPUTER SCIENCE (TOUR SITE)</p>
```

Comment:
what does tour site mean

Reply:
Please do not commit that piece of code it is just for testing purpose by mistake that code is pushed.